corehq/apps/repeaters/models.py

```diff
@@ -324,6 +261,13 @@ class FormRepeater(Repeater):
 
     """
 
+    class Formats(object):
+        formats = {
+            'form_xml': (FormRepeaterXMLPayloadGenerator, _('XML')),
+            'form_json': (FormRepeaterJsonPayloadGenerator, _('JSON')),
+        }
+        default_format = 'form_xml'
```

Comment:
would it make sense to let the `PayloadGenerator` handle what it is called / what it's format is?
That way you could just have: 
```py
formats = [PayloadGenerator1, PayloadGenerator2]
```
and have it be implicit that the first generator that is in the list be the default (i.e. it is the first one that shows up in the UI). 
This also makes the single-format case much easier, as it would just be a single thing in the list (and you don't need to remember that when there is only one format you use the word `generator` instead of `formats`).